objects/c932e8/script.lua

```diff
@@ -1,5 +1,36 @@
-function onLoad()
-    if not Global.getVar("gameStarted") then
+local initialized = false
+local serpent = nil
+local elementScanZone = nil
```

Comment:
i would probably suffix guid on there so it's clear that this isn't meant to hold an object reference